setup/cli-install.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+/**
+ * CLI install script for MODX
+ *
+ * Run and answer the questions. Also you can pre-define values:
+ * php www/setup/install.php --database=my_db --database_user=db_user_12 --language=ru
+ */
+
+if (PHP_SAPI != 'cli') {
+    exit('This file should be run through the command line interface!');
+}
+$path = dirname(__FILE__) . '/';
+$config = $path . 'config.xml';
+$mode = 'new';
+
+$variables = [
+    'database_type' => 'mysql',
+    'database_server' => [
+        'prompt' => 'Database server',
+        'default' => '127.0.0.1',
```

Comment:
Wouldn't `localhost` be a more appropriate default here? I seem to recall the regular setup defaults to that.

Reply:
localhost points to /var/run/mysql.sock, but in chroot environment it inaccessible. But 127.0.0.1 points to tcp port, that open in MySQL by default and works in chroot.

So, that is why it is by default in my script.